src/components/responsive-table/rows.tsx

```diff
@@ -44,8 +48,16 @@ const Rows = ({
     r.push(expandedRowComponent);
     return r;
   }, []);
-  let tbody = <tbody />;
-  if (tableRows.length) {
+
+  let tbody = <tbody></tbody>;
+
+  if (isLoading) {
+    tbody = (
+      <tbody>
+        <TableFiller columnsLength={visibleColumns.length} rowsLength={itemsPerPage} />
+      </tbody>
+    );
+  } else if (tableRows.length) {
     tbody = <tbody>{tableRows}</tbody>;
   }
   return tbody;
```

Comment:
you always return <tbody> but with different children

maybe instead of reassigning tbody, reassign children?

`
let content = <Text>fallback</Text>;

if (isLoading) content = <TableFiller />

if (tableRows.length) content = tableRows
return <tbody>{content}</tbody>`

Reply:
sounds reasonable